examples/http-client-oauth2-client-credentials-grant-type/http_client_oauth2_client_credentials_grant_type.md

```diff
@@ -1,6 +1,6 @@
 # HTTP client - OAuth2 client credentials grant type
 
-A client, which is secured with OAuth2 client credentials grant type can be used to connect to a secured service. The client is enriched with the `Authorization: Bearer <token>` header by passing the `http:OAuth2ClientCredentialsGrantConfig` for the `auth` configuration of the client.
+The `http:Client` can be secured with OAuth2 client credentials grant type authorization by enriching each request from the client with the `Authorization: Bearer <token>` header. The required configurations for this grant type can be specified in the `auth` field of the client configuration. Use this grant type when the client needs to access resources or services on behalf of itself rather than the user.
 
```

Comment:
Lets remove this part `Use this grant type when the client needs to access resources or services on behalf of itself rather than the user.`

Reply:
Fixed via https://github.com/ballerina-platform/ballerina-distribution/pull/3793/commits/373fd382554d45179c4ec44e8ea2280b4b8709b2